internal/cmd/group_flag.go

```diff
@@ -89,7 +89,10 @@ func parseDBSeedFlags(client *turso.Client) (*turso.DBSeed, error) {
 	}
 
 	if fromCSVFlag != "" {
-		return handleCSVFile(client, fromCSVFlag, csvTableNameFlag)
+		if csvSeparatorFlag != "" {
+			return handleCSVFile(client, fromCSVFlag, csvTableNameFlag, csvSeparatorFlag)
+		}
+		return handleCSVFile(client, fromCSVFlag, csvTableNameFlag, ",")
```

Comment:
```suggestion
		return handleCSVFile(client, fromCSVFlag, csvTableNameFlag, csvSeparatorFlag)
```

Flag already has a default value, so no need to treat the empty case.


When you need to handle it (not the case here), I suggest doing something like:
```go
		separator := ","
		if csvSeparatorFlag != "" {
			separator = csvSeparatorFlag
		}
		return handleCSVFile(client, fromCSVFlag, csvTableNameFlag, separator)
```